pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```
@@ -78,6 +83,51 @@ func (k *KubeASCheck) Run() error {
 	if err != nil {
 		k.Warn("could not collect API Server component status: ", err.Error())
 	}
+
+	var token string
+
+	if k.latestEventToken == 0 {
+		// Initialization: Checking if we previously stored the latestEventToken in a configMap
+		token, found, err := asclient.EventTokenFetcher()
+		if err != nil {
+			k.Warnf("Could not get the LatestEventToken from the eventtokendca ConfigMap: %s", err.Error())
+			token = ""
+		}
+		k.configMapAvailable = found
+		k.latestEventToken, _ = strconv.Atoi(token)
```

We shouldn't ignore the error here.